.vscode/settings.json

```diff
@@ -3,5 +3,10 @@
   "prettier.printWidth": 120,
   "prettier.semi": false,
   "prettier.singleQuote": true,
-  "tslint.enable": true
+  "tslint.enable": true,
+  "[typescript]": {
```

Comment:
@bmazzarol please do not change vscode settings: we try to add as less code as possible

Reply:
Sure mate.

I was having issues with vscode automatically re-ordering all the imports, but not in the order that the linter was happy with. Might have been my vscode setup, but thought it might have been useful for other committers.

Will remove.